### PR TITLE
Swift reference page

### PR DESCRIPTION
**Pull request by alex-tpom6oh**

### CODE REVIEW

File: pages/implementation/reference/swift.mdx [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+Avo functions usage consists of 4 steps.
+
+[//]: # (TODO For future: Here we can add link to the "Type safe code & unit tests" docs section when that's ready, something like:)
+[//]: # 'TODO Learn more about how Avo Functions can help decrease time spent on implementing analytics and increase data reliability in this guide.'
+
+#### Step 1. Include the Avo file
+
+##### Pull the generated code with the Avo CLI
+
+To get the Avo generated Kotlin file you must be a member of an Avo workspace with a Kotlin source. Ask for an invite from a colleague or create a <a href="https://avo.app/onboarding" target="_blank">new workspace</a>
```
Comment (alex-tpom6oh) [inline pages/implementation/reference/swift.mdx:26]: Swift

File: pages/implementation/reference/swift.mdx [hunk 2]
```diff
@@ -0,0 +1,208 @@
+---
+layout: 'guide'
+title: 'Swift'
```
Comment (logason) [inline pages/implementation/reference/swift.mdx:3]:
```suggestion
title: 'Avo Functions in Swift'
```

File: pages/implementation/reference/swift.mdx [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+---
+
+import Link from '../../../components/Link';
+
+### Platforms
+
+Avo can generate Swift code targeted at the following platforms
```
Comment (logason) [inline pages/implementation/reference/swift.mdx:10]:
```suggestion
Avo can code generate Avo Functions in Swift targeted at the following platforms
```

File: pages/implementation/reference/swift.mdx [hunk 4]
```diff
@@ -0,0 +1,208 @@
...
+```swift
+avo.signupStart(referral: "direct")
+```
+
+> Notice, that you are not passing the System property with the call. System properties are defined on the init step and then automatically included with all events.
+> You can update the system properties with `setSystemProperties` function.
+
+#### Step 4. Verify the implementation
+
+Use the <Link passHref href="/workspace/tracking-plan/implementation-status"><a>Implementation status</a></Link> in your Avo workspace and the <Link passHref href="/implementation/inspector/sdk/android"><a>Avo Inspector</a></Link> to verify that your implementation is correct.
```
Comment (logason) [inline pages/implementation/reference/swift.mdx:75]: Highlight debugger!

File: pages/implementation/reference/swift.mdx [hunk 5]
```diff
@@ -0,0 +1,208 @@
...
+
+### Reference
+
+#### Constructor
+
+```swift
+init(env: AvoEnv,
+    systemProprty0: Int, systemProperty1: Bool,
+    customDestDestination: AvoCustomDestination,
+    strict: Bool, noop: Bool);
```
Comment (logason) [inline pages/implementation/reference/swift.mdx:85]: hmw label them as optional here?

**Review by logason — APPROVED** [review]
Looks good! Had the same thoughts here as in the Kotlin docs

**Comment by hugihlynsson** [comment]
#### 📝Changed routes:


| [**…ation/reference/kotlin**](https://docs-qff5hnecv.avo-dev.app/implementation/reference/kotlin) | [**…tation/reference/swift**](https://docs-qff5hnecv.avo-dev.app/implementation/reference/swift) |
|:-:|:-:|
|<a href="https://docs-qff5hnecv.avo-dev.app/implementation/reference/kotlin">Screenshot of /implementation/reference/kotlin</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/b4ALr-qt_gWvkn05JOeKakw8wl9Poeigij9Jtevhrx4WbKKja_hVFeYAqcMUSI_xHgxcCFkSeUosEWFrgosY7w.png">(view full size)</a> |<a href="https://docs-qff5hnecv.avo-dev.app/implementation/reference/swift">Screenshot of /implementation/reference/swift</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/x5DsyUM-quNPfODk1ZjttlXmrNGu-b5Bi-6rKNTIW44BxRvqK2rOGtZzTDUX-S3xaGmQQYlMLwm-cXTazN1kvA.png">(view full size)</a> |



| [**…-programming-languages**](https://docs-qff5hnecv.avo-dev.app/implementation/supported-programming-languages) |
|:-:|
|<a href="https://docs-qff5hnecv.avo-dev.app/implementation/supported-programming-languages">Screenshot of /implementation/supported-programming-languages</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/kSlN_31dRhmJmmXQmRsQU8oMCuhnGrmUj3b-aLIesUIvvYkR6pY_Ax9yqC4touMHx_4tNR5ioiYqvRy5qR-qgw.png">(view full size)</a> |







Commit 6e8aa2a60437f5c1c724dccba085d1c750c637b3 (https://docs-qff5hnecv.avo-dev.app).

File: pages/implementation/reference/swift.mdx (hunk 5 above)
Comment (alex-tpom6oh) [reply pages/implementation/reference/swift.mdx]: added a comment similar to one in kotlin

**Pull request merged**